docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -185,12 +181,14 @@ To verify that the cluster is operating correctly, use the [monitoring mixin das
 
 ## Migrating to Grafana Mimir using Helm
 
-You can update to the Grafana Mimir Helm chart from the Cortex Helm chart.
+You can migrate to the Grafana Mimir Helm chart (`grafana/mimir-distributed` v3.1.0) from the Cortex Helm chart
+(`cortex-helm/cortex` v1.7.0).
 
 ### Before you begin
 
-- Ensure that you are running the v1.4.0 release of the Cortex Helm chart.
+- Ensure that you are running the v1.7.0 release of the Cortex Helm chart.
 - Ensure that you are running ingesters using a Kubernetes StatefulSet.
+- We are using `yq` v4 to parse and mofidy yaml files.
```

Comment:
@dimitarvdimitrov I added yq version requirement here.

Reply:
i was referring to the two yq programs

https://github.com/kislyuk/yq
and
https://github.com/mikefarah/yq

iirc they have a different API

Reply:
I always use mikefarah yq and just got to know there is another yq implementation with sizeable github stars. I will update the guide with recommendations on which yq to use. I tested both have same api to get field and to delete fields. However kilyuk yq is spitting out json (which is subset of yaml anyway) instead of yaml.